routes/lang/en-application.json

```diff
@@ -148,7 +148,8 @@
   "app.projectsettings": "Project settings",
   "app.projectcontentbody": "This is a text component",
   "app.configurationsettings": "Configuration settings",
-  "app.themepicker": "Theme picker",
+  "app.themesettings": "Theme settings",
```

Comment:
Probably need to revert this line:

![picker](https://user-images.githubusercontent.com/922987/52484513-6dec0900-2bae-11e9-832a-d531d2bad428.png)